### DISCUSSION TITLE
Can CodePath executable be in a subfolder?

### DISCUSSION THREAD

**Discussion by jphalip** [elgatosf/elgato#61] [Help] [answered]
Hi,

I've built an executable binary for my plugin. It looks like the binary file, as specified in the "CodePath" attribute, must be in the same folder as the manifest file. I've tried putting it in a subfolder but that doesn't seem to work. I've only managed to make it work by putting it in the same folder as the manifest. Is that a requirement?

**Comment by ChekTek** [comment, marked as answer]
Hi @jphalip,

It seems to be working for me, although I'm testing a javascript plugin. 
```
	"CodePath": "plugin/app.html",
```

What version of Stream Deck are you using?

**Reply by GeekyEggo** [reply]
Having tested with Govee Controller, native plugins with nested entry points are also fine.
